python/jupyter-notebooks/ohsome_green_region.ipynb

```diff
@@ -0,0 +1,794 @@
+      "text/plain": [
+       "<Figure size 648x504 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# define and plot request/path\n",
+    "def area():   \n",
+    "    res = elements(\"/area/density/groupBy/boundary\"                  #endpoint\n",
+    "         ,filter=filter_req\n",
+    "         ,bpolys=regions                                             #spatial boundary\n",
```

Comment:
Here again regarding comments: you don't have to comment too much, especially stuff that should be understandable just through the code. The code here `bpolys=regions` alone should already be enough self-explanatory that you won't need to add a comment saying what this line represents.

In general, it's mostly enough to add 1-2 lines of comments at the top of a function and then only on very specific code lines, where a bit more explanation is needed, because for example something rather complex is happening here.